engine/access/rpc/connection/connection_test.go

```diff
@@ -188,7 +200,11 @@ func TestProxyExecutionAPIConnectionReuse(t *testing.T) {
 
 	req := &execution.PingRequest{}
 	expected := &execution.PingResponse{}
-	en.handler.On("Ping", testifymock.Anything, req).Return(expected, nil)
+	en.handler.
+		On("Ping",
+			testifymock.Anything,
+			testifymock.AnythingOfType("*access.PingRequest")).
```

Comment:
Might need `*execution.PingRequest` here and below:
```suggestion
			testifymock.AnythingOfType("*execution.PingRequest")).
```